promptulate/preset_roles/prompt.py

```diff
@@ -41,6 +41,19 @@
 
 """  # noqa: E501
 
+# 250+ token
+PRESET_SYSTEM_PROMPT_ZhiPU = """
```

Comment:
Can we remove it？It's repeat and useless. One system prompt preset is enough.

Reply:
ok